charts/neuvector/1.8.901+up1.8.9/Chart.yaml

```diff
@@ -0,0 +1,28 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: NeuVector
+  catalog.cattle.io/kube-version: '>=1.18.0-0'
+  catalog.cattle.io/namespace: neuvector-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/rancher-version: '>= 2.6.4-0'
+  catalog.cattle.io/release-name: neuvector
+  catalog.cattle.io/type: cluster-tool
+  catalog.cattle.io/ui-component: neuvector
+  catalog.cattle.io/upstream-version: 1.8.9
+apiVersion: v1
+appVersion: 4.4.3
+description: Helm chart for NeuVector's core services
+home: https://neuvector.com
+icon: https://avatars2.githubusercontent.com/u/19367275?s=200&v=4
+keywords:
+- security
+kubeVersion: '>=1.18.0-0'
+maintainers:
+- email: support@neuvector.com
+  name: becitsthere
+name: neuvector
+sources:
+- https://github.com/neuvector/neuvector
+- https://github.com/neuvector/manager
+- https://github.com/neuvector/scanner
+version: 1.8.901+up1.8.9
```

Comment:
@aiyengar2 should these be 100.0.0+up1.8.9 ? Its a brand new chart, so we could also start at 1.0.0. But I think it should be 100 to be normal.